CONTRIBUTING.md

```diff
@@ -68,6 +68,12 @@ follow work through the issue/pull request process.
 5. **Open a pull request into `develop`.** Our main branch is `develop`, and
    this is where we want new work to be merged into. `master` is a release
    branch only.
+6. **Update `CHANGELOG.md` in your branch.** Add the pull-request number and 
+   link, and a brief, one line description under the next tag title. Make sure 
+   the reason for the change can be understood. If this change has a chance of 
+   making an existent project's build fail, or cause unexpected changes in the 
+   output, group it under the "Possible breaking changes" title. New features
```

Comment:
I don’t think this makes a clear statement on what a breaking change is. Basically, anything that would alter the visual output of an existing project (not including custom extensions) after an update and isn’t a bugfix is a breaking change. It also should be made clear that due to semver this means the need for a new major release. I think that is an important fact to add since many people don’t think about this beforehand and just go by "this is a little break but it’s easy to fix for users", which is a fallacy that we won’t go here at inuitcss. We want to provide a well working toolset that is maintained in a solid way and does follow semver as it’s intended.